src/platform/Linux/PlatformManagerImpl.cpp

```diff
@@ -148,6 +150,44 @@ CHIP_ERROR PlatformManagerImpl::_InitChipStack()
     return err;
 }
 
+CHIP_ERROR PlatformManagerImpl::_GetCurrentHeapFree(uint64_t & currentHeapFree)
+{
+    struct mallinfo mallocInfo = mallinfo();
```

Comment:
https://man7.org/linux/man-pages/man3/mallinfo.3.html

says that mallinfo is deprecated in favor of mallinfo2. Do we care?

Reply:
I also noticed this comments, but mallinfo2() function was added in glibc 2.33. But Ubuntu 20.04 still use 
Ubuntu GLIBC 2.31-0ubuntu9.2) 2.31